test/parallel/test-tick-processor.js

```diff
@@ -15,52 +16,95 @@ if (common.isAix) {
 }
 
 common.refreshTmpDir();
-process.chdir(common.tmpDir);
+
+const tests = [];
+
+const LOG_FILE = path.join(common.tmpDir, 'tick-processor.log');
+const RETRY_TIMEOUT = 750;
+
 // Unknown checked for to prevent flakiness, if pattern is not found,
 // then a large number of unknown ticks should be present
-runTest(/LazyCompile.*\[eval\]:1|.*%  UNKNOWN/,
-        `function f() {
+tests.push({
+  pattern: /LazyCompile.*\[eval\]:1|.*%  UNKNOWN/,
+  code: `function f() {
            for (var i = 0; i < 1000000; i++) {
              i++;
            }
            setImmediate(function() { f(); });
          };
-         setTimeout(function() { process.exit(0); }, 2000);
-         f();`);
+         f();`
+});
+
 if (common.isWindows ||
     common.isSunOS ||
     common.isAix ||
     common.isLinuxPPCBE ||
     common.isFreeBSD) {
   common.skip('C++ symbols are not mapped for this os.');
-  return;
+} else {
```

Comment:
Did you mean to include the rest of the tests inside this `else` block? Either that or the `return` keyword has to be restored, I would think...